docs/ops-recipe/projectQuotaIsRaisingTheLimit.md

```diff
@@ -0,0 +1,20 @@
+---
+
+title: "ProjectQuotaIsRaisingTheLimit"
+
+---
+
+This alert fires because a Harbor project is nearing the limit of its capacity.
+
+Projects are created with a default capacity value of `-1` which means it has unlimited storage. If your projects are continuously approaching their limit, you may want to consider switching the limit to this.
+
+The alert should show the name of the project that is getting full. You can manage and adjust the projects capacity in the Harbor UI. 
```

Comment:
Storage mean S3? Also does it make sense to handle changes on the UI? Does it get persisted somewhere or can we set these from code instead atm?